pkgs/top-level/all-packages.nix

```diff
@@ -4128,6 +4128,8 @@ with pkgs;
     protobuf = protobuf3_21;
   };
 
+  anbox-postmarketos-image = callPackage ../os-specific/linux/anbox/postmarketos-image.nix { };
```

Comment:
Since this is only used for tests we should probably move it to passthru.

Reply:
Done.